sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsRecordedTestSanitizer.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Text;
+
+namespace Azure.Core.Testing
+{
+    public class TextAnalyticsRecordedTestSanitizer : RecordedTestSanitizer
+    {
+        public override void SanitizeHeaders(IDictionary<string, string[]> headers)
+        {
+            const string key = "Ocp-Apim-Subscription-Key";
```

Comment:
we could call `base.SanitizeHeaders(headers);` first, just in case there is something now (or later) that will get sanitized automatically

Reply:
Deleted it accidentally. Nice catch.